src/components/functions/savePlayer.tsx

```diff
@@ -0,0 +1,20 @@
+import showPlayer from './showPlayer.tsx'
```

Comment:
functions folder should not be part of components folder. it should be on the same level as components folder in the structure